index.html

```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Gowani Pump Supply</title>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Google Fonts -->
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="bower_components/bootstrap/dist/css/bootstrap.min.css">
+
+    <!-- Style Sheet for Sticky Bootstrap Footer -->
+    <link href="css/sticky-footer.css" rel="stylesheet">
+
+    <!-- Animate CSS -->
+    <link rel="stylesheet" href="css/animate.css">
+
+    <!-- Custom Style Sheets -->
+    <link rel="stylesheet" href="css/style.css">
+
+  </head>
+
+  <body>
+
+  <nav class="navbar navbar-toggleable-sm navbar-inverse bg-faded fixed-top">
```

Comment:
A static navbar works and looks better than `fixed-top` - it'll stay at the top and not scroll with the user. It'll also let you close #4.